rules/integrations/kubernetes/execution_pod_created_in_kube_namespace.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2022/05/23"
+maturity = "production"
+updated_date = "2022/05/23"
+integration = "kubernetes"
+
+[rule]
+author = ["Elastic"]
+description = """
```

Comment:
can you run `python -m detection_rules toml-lint -f <this-file>` on this

Reply:
I couldn't get this to work locally, so I just reran the jobs here. That should do the same thing right?

Reply:
Were you getting an error?

Reply:
Hey, not so much an error but an annoyance. I was missing lots of python modules so kept running it and then having to install a module. I figured it'd be faster to just re-run the jobs here and get my system set up properly to run the jobs locally later.